### ISSUE TITLE
PNS neuron doest automatically classify stuff

### ISSUE THREAD

**Issue by shawntanzk** [obophenotype/cell-ontology#1537]
Example: 
sensory neuron of spinal nerve - equiv: 'sensory neuron' and ('part of' some 'spinal nerve')
spinal nerve - subclass: 'part of' some 'peripheral nervous system'

PNS neuron - equiv: neuron and ('has soma location' some 'peripheral nervous system')

but sensory neuron of spinal nerve is not inferred as a PNS neuron 

Is this due to the use of has soma location instead of part of (we 100% cannot use part of for PNS neuron)? If so, do we need to look into manually asserting some stuff as PNS neuron - my worry is current when querying, it might not show stuff we want to show. I get that it is hard as PNS is technically not in CNS, and negative is tricky to express in ontologies. 

Note: CNS neuron seems to use reasoner well - im guessing in part to do with stuff from uberon 

Related to https://github.com/obophenotype/cell-ontology/issues/1069

**Comment by paolaroncaglia** [comment]
I agree with @shawntanzk that we'd want to be able to query PNS neuron types somehow. I know that having to assert manually is not ideal and prone to forgetfulness, so ideally a more sustainable solution should be found. Otherwise, hopefully it's not too many children that need asserting.

**Comment by dosumis** [comment]
I suggest using 'has soma location' in all cases.  What's the argument against this?

**Comment by shawntanzk** [comment]
Somehow didn't think of that >.< will make the changes to all using 'has soma location' 
I'm guessing there's no way to easily cover all cases out of the CNS as PNS -> we can't say that if the cell isn't in Brain or Spinal cord, it is PNS, as this would make terms that do not have location details PNS which would be problematic. What I can do instead is go through manually to classify things like Cardiac neuron and Kidney nerve cell as PNS neuron

**Comment by shawntanzk** [comment]
@dosumis  - just tried to make the change - does this mean we change UBERON term
spinal nerve - subclass: 'part of' some 'peripheral nervous system' -> change 'part of' to 'has soma location' too? This is required to get the classification there right

**Comment by dosumis** [comment]
> spinal nerve - subclass: 'part of' some 'peripheral nervous system' -> change 'part of' to 'has soma location' too? 

No.  Domain of 'has soma location' is neuron.   https://en.wikipedia.org/wiki/Soma_(biology)

> This is required to get the classification there right.

Shouldn't be.  We have this property chain:

`'has soma location' o 'part of' SubPropertyOf: 'has soma location'`
